osu.Game/GameModes/Play/Mania/ManiaHitRenderer.cs

```diff
@@ -17,13 +16,10 @@ public class ManiaHitRenderer : HitRenderer<ManiaBaseHit>
         public ManiaHitRenderer(int columns = 5)
         {
             this.columns = columns;
+            Converter = new ManiaConverter(columns);
         }
 
-        protected override List<ManiaBaseHit> Convert(List<HitObject> objects)
-        {
-            ManiaConverter converter = new ManiaConverter(columns);
-            return converter.Convert(objects);
-        }
+        protected override HitObjectConverter<ManiaBaseHit> Converter { get; }

```

Comment:
just create the converter here like other classes, rather than storing a local one for consistency?

`Converter => new ManiaConverter(columns)`